pages/common/luac.md

```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua byte code compiler.
```

Comment:
Not sure that there should be a space in `bytecode`.

Reply:
My spellchecker underlined it but on wiki it says bytecode. I'll change it